packages/utils/src/cryptography/snip12.cairo

```diff
@@ -32,7 +32,7 @@ pub trait OffchainMessageHash<T> {
     fn get_message_hash(self: @T, signer: ContractAddress) -> felt252;
 }
 
-impl StructHashStarknetDomainImpl of StructHash<StarknetDomain> {
```

Comment:
Does this need to be pub? It should be enough to import the trait if needed, because the impl would be automatically in scope.